lambdatest-espresso/src/main/java/io/github/lambdatest/LTApp.java

```diff
@@ -0,0 +1,96 @@
+package io.github.lambdatest;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import io.github.lambdatest.client.HttpClient;
+import io.github.lambdatest.utils.Utils;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import io.github.lambdatest.constants.Constants;
+
+public class LTApp {
+
+    private HttpClient httpClient;
+    public static String LOGLEVEL = "info";
+    private static boolean DEBUG = LOGLEVEL.equals("debug");
+    private static String LABEL = "[\u001b[35m" + ("lambdatest-espresso") + "\u001b[39m]";
+    public static Boolean ignoreErrors = true;
+
+
+    public LTApp() {
+        this.httpClient = new HttpClient();
```

Comment:
whats the use of this `httpClient` instance?
In Utils class, we are initiating client instance again
```
Utils utils = new Utils();
```